prince-of-versions/src/main/java/co/infinum/princeofversions/helpers/parsers/JsonVersionConfigParser.java

```diff
@@ -95,6 +95,16 @@
     public static final String META = "meta";
 
     /**
+     * Last minSdk key
+     */
+    public static final String LAST_MIN_SDK = "minimum_version_min_sdk";
```

Comment:
I don't like naming here and in all other classes where that field's value is propagated. Probably something like `MIN_VERSION_MIN_SDK` should better reflect that this field represents `MIN_SDK` of minimum/mandatory version.